apps/hiring-kanban/src/components/CardForms/CardHistory.tsx

```diff
@@ -24,7 +32,46 @@ function TitleComponent({ value }: CellProps<TCardHistoryCard>) {
   return <div className="text-sm truncate">{value || 'Untitled'}</div>;
 }
 
+function DoubleIconDisabled({
+  title,
+  updatePreview,
+  leftIcon,
+  rightIcon,
+}: {
+  title: string;
+  updatePreview: () => void;
+  leftIcon: boolean;
+  rightIcon: boolean;
+}) {
+  return (
+    <button type="button" title={title} disabled onClick={() => updatePreview}>
+      {leftIcon && (
+        <ChevronDoubleLeftIcon
+          aria-hidden="true"
+          className="absolute -mt-60 ml-10  h-8 w-8 text-stone-400 cursor-not-allowed"
```

Comment:
something that is probably worth looking into is the ‘top, left, right, bottom’ attributes for css. They are used with absolute (and static/fixed/sticky) positioned elements rather than using margin